tcms/signals.py

```diff
@@ -175,3 +176,34 @@ def handle_comments_pre_delete(sender, **kwargs):
     instance = kwargs['instance']
 
     get_comments(instance).delete()
+
+
+def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
+    '''
+        Send email updates to assignee after they've been
+        assigned a bug on bug creation.
+    '''
```

Comment:
Use 3 double quotes, this is the convention we use. Pylint was supposed to catch this but for some reason it didn't which is strange.